components/CreateEditDialog.tsx

```diff
@@ -0,0 +1,109 @@
+import { observer } from "mobx-react-lite";
+import { useContext, useState } from "react";
+import { v4 as newUuidV4 } from "uuid";
```

Comment:
typescript complains on this:
Could not find a declaration file for module uuid.

Reply:
Have you run `npx expo install`? It works locally for me.